python/ray/scripts/scripts.py

```diff
@@ -395,6 +396,12 @@ def debug(address):
     default=False,
     help="provide this argument for the head node",
 )
+@click.option(
+    "--start-gcs",
```

Comment:
```suggestion
    "--gcs-only",
```
? start-gcs sounds confusing to me because --head also start gcs.